article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -0,0 +1,174 @@
+@import layout.ContentWidths.MainMedia
+@import model.content.MediaAtom
+@import common.Localisation
+@(articleModel: ArticlePage, recipes: Seq[model.content.RecipeAtom])(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
+
+@import common.LinkTo
+@import views.BodyCleaner
+
+@schemaType(page: ArticlePage) = @{ page.article.metadata.schemaType }
+@bodyType(page: ArticlePage) = @{ "articleBody" }
+
+@defining(articleModel.article) { article =>
+    <article id="article" data-test-id="article-root"
+            class="content content--recipes"
+            itemscope itemtype="@schemaType(articleModel)" role="main" >
+    <meta itemprop="mainEntityOfPage" content="@LinkTo(article.metadata.url)">
+    <div itemprop="publisher" itemscope itemtype="https://schema.org/Organization">
+        <meta itemprop="name" content="The Guardian">
+        @fragments.logo(amp = false)
+    </div>
+
+    <div class="recipes__main">
+        <div class="recipe__images-wrapper">
+            <div class="recipe__label">
+                <h2>food &amp; drink</h2>
+            </div>
+            <div class="recipe__image__wrapper recipe__image__wrapper--is-displayed">
+            @fragments.image(picture = defaultImage,
+                classes = Seq("recipe__image", "fadeIn"),
+                imageAltText = "main alt text here",
```

Comment:
Default images have alt text set if available, however recipe atoms' image assets do not have a caption fields set and so these are being set to "". Thanks @gidsg for your advice on this.